### DISCUSSION TITLE
都有问题

### DISCUSSION THREAD

**Discussion by kingwoonyu** [natfrp/wiki#277] [使用问题]
显示这个
E Tunnel/Arma3up [22/526227/8879] work connection closed before response StartWorkConn message: EOF

**Comment by Yue-cn** [comment]
## 不会搞就请您更换节点，谢谢。
## 这一般是与 FRPS 连接不稳。

**Comment by kingwoonyu** [comment]
就一个beta有用，艹




??风早沫?
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "natfrp/wiki"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年2月11日(星期六) 下午4:33
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [natfrp/wiki] 都有问题 (Discussion #277)





 
不会搞就请您更换节点，谢谢。
